### ISSUE TITLE
libfill_lib.so: undefined reference to `veraInit()'

### ISSUE THREAD

**Issue by ftpronk** [SX-Aurora/veda#8]
I'm getting a linking error in my test project, trying to use VERA:
```
[ 85%] Linking CXX executable veda_test
libfill_lib.so: undefined reference to `veraInit()'
```
I understand I should be linking my host library with `libvera.so` if I'm going to use VERA functionality.  For the VEDA functionality, CMake defines VEDA_LIBRARY:
```
grep -r "FIND_LIBRARY"
veda-0.9.5.1/cmake/FindVE.cmake:  FIND_LIBRARY(VEDA_LIBRARY "libveda.so" "libveda.a" PATHS "${VEDA_DIR}/lib64")
```
but I see no equivalent for the VERA_LIBRARY variable.

Am I doing something wrong?

**Comment by mergian** [comment]
Thanks for reporting. Added the missing line: https://github.com/SX-Aurora/veda/blob/69e2660f61cf6ee9a98709b4d8781fe59bd4c1d5/cmake/FindVE.cmake#L4

To fix your problem before we push out a new release, you can either compile VEDA yourself, or just copy the line into your CMakeLists.txt.

**Comment by mergian** [comment]
Fixed in v0.9.5.2
